index.html

```diff
@@ -7,26 +7,60 @@
   <title>Awesome Books</title>
   <link rel="stylesheet" href="stylesheets/reset.css">
   <link rel="stylesheet" href="stylesheets/style.css">
+  <script src="scripts/luxon.min.js"></script>
   <script src="scripts/app.js"></script>
 </head>
 <body>
   <header>
-    <h1 id='main-title'>All Awesome Books</h1>
+    <div class="navigation">
+      <nav class="topnav">
+        <ul class="links">
+          <div class="part-one">
+            <li id="logo">Awesome Books</li>
+          </div>
+          <div class="part-two">
+            <li class="link-item"><a href="#" id="list-btn">List</a></li>
+            <li class="link-item"><a href="#" id="add-new-btn">Add new</a></li>
+            <li class="link-item"><a href="#" id="contact-btn">Contact</a></li>
+          </div>
+        </ul>
+      </nav>
+    </div>
+  <hr id="separator"><br><br><br>
+  <p id="date-string"></p>
   </header>
   <main>
   <div id="main-list">
+  <h1 id='main-title'>All Awesome Books</h1>
   </div>
   <div>
     <h2 id="second-title">Add new book:</h2>
     <br>
-      <form action="/index.html">
+      <form id="form-field" action="/index.html">
         <label for="title"></label><br>
         <input type="text" id ="title" name="title" placeholder="Tiltle"><br><br>
         <label for="author"></label><br>
         <input type="text" id="author" name="author" placeholder="Author"><br><br>
```

Comment:
- [ ] Please make the input field required I still can add an empty book to the list.